lib/csv_row_model/export.rb

```diff
@@ -0,0 +1,28 @@
+module CsvRowModel
+  # Include this to with {Model} to have a RowModel for exporting to CSVs.
+  module Export
+    extend ActiveSupport::Concern
+
+    included do
+      attr_reader :source_model
+
+      self.column_names.each do |column_name|
+        define_method(column_name) do
```

Comment:
I need to add some documentation here and modify the readme. This is safe to override in the case whe the column name doesn't correspond to a direct attribute of the model.